composer.json

```diff
@@ -111,15 +111,15 @@
 	"require-dev": {
 		"ext-pdo_sqlite": "*",
 
-		"matthiasnoback/symfony-config-test": "dev-phpunit-10",
+		"matthiasnoback/symfony-config-test": "~4.3.0",
```

Comment:
Before merging this, please change to 5.0 because the 4.3.x release is not compatible with PHPUnit 10